cni/pkg/repair/repair.go

```diff
@@ -145,7 +160,11 @@ func (bpr BrokenPodReconciler) deleteBrokenPod(pod v1.Pod) error {
 		return nil
 	}
 	log.Infof("Pod detected as broken, deleting: %s/%s", pod.Namespace, pod.Name)
-	return bpr.client.CoreV1().Pods(pod.Namespace).Delete(context.TODO(), pod.Name, metav1.DeleteOptions{})
+	err := bpr.client.CoreV1().Pods(pod.Namespace).Delete(context.TODO(), pod.Name, metav1.DeleteOptions{})
+	if err == nil {
+		bpr.Metrics.PodsRepaired.Increment()
```

Comment:
It feels weird to get a metric for success and not have a corresponding one for failures. Failures are arguable more important?

Also, very nit, the pattern of `err == nil` is a bit odd
```go
if err := bpr.client.CoreV1().Pods(pod.Namespace).Delete(context.TODO(), pod.Name, metav1.DeleteOptions{}); err != nil {
return err
}
increment()
return nil
```

seems more standard to me